tests/dummy/app/pods/icons/controller.js

```diff
@@ -1,6 +1,6 @@
+import _ from 'lodash'
```

Comment:
I don't believe we've recorded the preferred ordering of imports (I had been ordering alphabetically on the package being imported from) - probably worth stating in the style guidelines.

Is this something we could eslint --fix?